src/builtins.c

```diff
@@ -524,6 +523,11 @@ JL_CALLABLE(jl_f_isdefined)
     }
     if (nargs != 2) {
         JL_NARGS(isdefined, 1, 1);
+        jl_depwarn("`isdefined(:symbol)` is deprecated, "
```

Comment:
delete every line containing the deprecated method:
```
grep -RI current_module . | wc -l
62
```
and then refactor the `eval` methods to reflect the fact that this information no longer needs to be maintained